### ISSUE TITLE
New update button

### ISSUE THREAD

**Issue by TuringVerified** [Cobweb-Aclevo/celos#47]
Adding an update button in [this menu](https://user-images.githubusercontent.com/40866259/134129754-a41dfe89-cbc0-49a3-976d-3e23d0c809ad.png) and removing the annoying update prompt to make the OS more beginner friendly.

**Comment by moeenio** [comment]
Problem : what do we do with ubunti's updater? Since it doesn't support flatpak it obviously won't be used here, but should it stay on the system?

**Comment by TuringVerified** [comment]
I do not think it should. That'd be in conflict with CelOS's own update functionality.

**Comment by moeenio** [comment]
Yes, but removing it might break some things.

**Comment by moeenio** [comment]
I say this is to look at for CelOS-beta8.

**Comment by TuringVerified** [comment]
Okay then

**Comment by TheOceanBreeze** [comment]
There is a package to let Gnome handle updating Flatpaks, but it may still require restarts to update apt packages unless we can disable that.

If we can't use Gnome's Software Center, we could look into other software to update the system.

Also, how do we feel about automatic updates as opposed to manual updates?

**Comment by moeenio** [comment]
> There is a package to let Gnome handle updating Flatpaks

You're probably talking about `gnome-software`.
I haven't thought about it at first and didn't look back at this issue, but we might be able to just fully replace ubuntu's updater with it.

It looks like restart to update can't really be disabled and I actually think it's a good idea: see [this post from a Fedora forum](https://ask.fedoraproject.org/t/gnome-software-center-wants-me-to-restart-to-install-updates/1283/7), I don't see a reason why what is said wouldn't apply to ubuntu.

I personally don't like automatic updates, I don't appreciate something randomly using CPU in the background - however I use old hardware, and they're so easy to enable or disable in gnome-software that we could have them on by default

**Comment by TheOceanBreeze** [comment]
> > There is a package to let Gnome handle updating Flatpaks
> 
> You're probably talking about `gnome-software`. I haven't thought about it at first and didn't look back at this issue, but we might be able to just fully replace ubuntu's updater with it.
> 
> It looks like restart to update can't really be disabled and I actually think it's a good idea: see [this post from a Fedora forum](https://ask.fedoraproject.org/t/gnome-software-center-wants-me-to-restart-to-install-updates/1283/7), I don't see a reason why what is said wouldn't apply to ubuntu.
> 
> I personally don't like automatic updates, I don't appreciate something randomly using CPU in the background - however I use old hardware, and they're so easy to enable or disable in gnome-software that we could have them on by default

I agree with using this solution for our distro.
